dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/Collections/CollectionConformanceTests.cs

```diff
@@ -103,7 +113,17 @@ private async Task CreateCollectionIfNotExistsMoreThanOnce<TRecord>()
         try
         {
             Assert.True(await collection.CollectionExistsAsync());
-            Assert.True(await fixture.TestStore.DefaultVectorStore.ListCollectionNamesAsync().ContainsAsync(collection.CollectionName));
+
+#pragma warning disable MEVD9000 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+            var collectionMetadata = collection.GetService(typeof(VectorStoreRecordCollectionMetadata)) as VectorStoreRecordCollectionMetadata;
+#pragma warning restore MEVD9000 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+
+            Assert.NotNull(collectionMetadata);
+            Assert.NotNull(collectionMetadata.VectorStoreSystemName);
+            Assert.NotNull(collectionMetadata.VectorStoreName);
```

Comment:
This fails for e.g. Weaviate, where there's no VectorStoreName (I guess we're not running these integration tests in CI?).